### PR TITLE
Amended prerequisite for JDK in kafka getting started guides.

### PR DESCRIPTION
**Pull request by AedinC**
Removed and amended JDK prerequisite in kafka getting started guides.

### CODE REVIEW

File: docs/kafka/kcat-kafka/README.adoc [hunk 1]
```diff
@@ -73,8 +73,7 @@ ifndef::community[]
 endif::[]
 //* You have a subscription to {product-long-kafka}. For more information about signing up, see *<@SME: Where to link?>*.
 * You have a running Kafka instance in {product-kafka}.
-* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
-* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
+* https://adoptopenjdk.net/[JDK^] 11 or later is installed. The latest LTS version of OpenJDK is recommended.
```
Comment (sterobin) [inline docs/kafka/kcat-kafka/README.adoc:76]: Need parentheses for consistency: `(The latest ... )`

**Review by sterobin — COMMENTED** [review]
@AedinC Good with me. Just a small correction.

File: docs/kafka/kcat-kafka/README.adoc (hunk 1 above)
Comment (AedinC) [reply docs/kafka/kcat-kafka/README.adoc:76]: Thanks Stetson. Will correct.

**Comment by wtrocki** [comment]
Looks good.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**